<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeRangeEditor2</class>
 <widget class="QDialog" name="TimeRangeEditor2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TimeRangeEditor2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Begin:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dt_begin"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_snap">
       <property name="text">
        <string>|&lt;- Snap</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_has_end">
       <property name="text">
        <string>End:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dt_end"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DateTimeSelector" name="w_slider" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_begin">
       <property name="text">
        <string>Go To Begin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_now">
       <property name="text">
        <string>Go To Now</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DateTimeSelector</class>
   <extends>QWidget</extends>
   <header>datetimeselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TimeRangeEditor2</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TimeRangeEditor2</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
